os/arch/xtensa/src/esp32/esp32_i2c.c

```diff
@@ -144,6 +150,7 @@ struct esp32_i2c_priv_s {
 
 	int xfer_speed;
 	uint32_t slave_addr;
+	uint8_t slave_addr_bits;
```

Comment:
10bits slave device access feature  is not related with  debuging long data reading fail issue,
spliting it one more PR may be clear.

Reply:
yes, thanks.